experiment_runner_node:
  ros__parameters:
    experiment_runner_adapter:
      start_position: [ 3., 3., 0. ]
base_movement:
  ros__parameters:
    reference_points: [ 3., 3., 0. , 3., 3., 3.1415926535]
    rotation_angle: 10
